src/assertions/eq.ts

```diff
@@ -37,6 +37,8 @@ export function eq(ctx: any, task: any, value: any, condition: any) {
       throw new AssertionFailed('eq', JSON.stringify(condition), JSON.stringify(value))
     }
   } else if (value !== condition) {
-    throw new AssertionFailed('eq', JSON.stringify(condition), JSON.stringify(value))
+    if (isNaN(Date.parse(value)) || isNaN(Date.parse(value)) || Date.parse(value) !== Date.parse(condition)) {
```

Comment:
Here it seems that `isNaN(Date.parse(value))` is duplicated. I think that the intention here was to check that the `condition` is a date too.

Reply:
Yup, that's right. Thank you!